integreat_compass/cms/templates/offers/offer_report.html

```diff
@@ -15,6 +15,7 @@ <h1 class="heading">
                 Report offer "{{ offer_title }}"
             {% endblocktranslate %}
         </h1>
+        <p>{% translate "Please note: This report can be seen by both the offer provider and board members." %}</p>
```

Comment:
```suggestion
        <p>{% translate "Please note: You will stay anonymous. The report will be seen by both the offer provider and board members." %}</p>
```

(or something along the lines - I think the anonymous part is important because otherwise it could be misinterpreted as "better watch your tongue, we'll show this to lots of people!" 😄)